mariadb.spec

```diff
@@ -219,7 +239,7 @@ BuildRequires:    perl-interpreter
 BuildRequires:    perl-generators
 %endif
 # Some tests requires python
-BuildRequires:    python
+BuildRequires:    python3
```

Comment:
This should be under rhel > 7 conditional

Reply:
fixed, thx.